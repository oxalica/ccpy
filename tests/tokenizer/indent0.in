  begin

a
    b
    c

      d
    e
        # Empty lines should be skipped without counting its indentation
      f
# It should produce 2 `Dedent` and 1 `Indent` here
  g
  h

  # f
#

a
  b
    # EOF may also produce indentations
    c