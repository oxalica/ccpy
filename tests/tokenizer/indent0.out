Indent
Name("begin")
Newline
Dedent
Name("a")
Newline
Indent
Name("b")
Newline
Name("c")
Newline
Indent
Name("d")
Newline
Dedent
Name("e")
Newline
Indent
Name("f")
Newline
Dedent
Dedent
Indent
Name("g")
Newline
Name("h")
Newline
Dedent
Name("a")
Newline
Indent
Name("b")
Newline
Indent
Name("c")
Dedent
Dedent
End
